### DISCUSSION TITLE
Новые анимации

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#582] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/shifter/32/1377_2.png) [Shifter](https://forum.octothorp.team/u/Shifter) ([#10103](https://forum.octothorp.team/p/10103))

Предложение со старого форума
 Анимации лёжа/на земле:
![image](https://forum.octothorp.team/uploads/default/original/2X/7/7174cad262d4d3ac5f6b02173d71e00ab0920a37.jpeg)
 Название: D1Town05 Winston Down; D1Town05 Wounded Idle 1; D1Town05 Wounded Idle 2.

Анимация стоя
![image](https://forum.octothorp.team/uploads/default/original/2X/c/cd6e84b751e454199b9e747b3e24e035139d9608.jpeg)
 Название: LineIdle02; LineIdle04.

- Добавить
- Нет

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/4cb/32/176_2.png) [4CB](https://forum.octothorp.team/u/4CB) ([#10110](https://forum.octothorp.team/p/10110))


Анимации лежа было бы классно(но еще бы добавить к твоим из скрина анимку животом на земле и с руками за спиной или за головой). А вот анимки стоя очень уж похожи на “Скрестить руки” и “Руки на пояс”, так что думаю лучше добавить другие.

**Comment by Ggman77** [comment]
Я за, добавит отмосверности и реализма!

**Comment by LENFYS** [comment]
За

**Comment by travka4000** [comment]
я за, даже из-за того когда коп попросит лечь на землю ты поставишь себе эту анимацию, а не будешь сидеть на корточках как гопник
